ruby-challenges.rb

```diff
@@ -32,3 +63,25 @@
 # Expected output: 'LEARN is not a palindrome'
 palindrome_tester3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+# Psuedo Code
+# Create a method called palin_no_yes
+# input will be a str
+# declare two varibles to store: 1. to store the origal str just lowercased. 2. will store a lowercase and reversed str.
+# using a conditional compare if both varibles are equal
+# if the strings are equal then return "#{str} is a palindrome" 
+# if the strings are not equal then return "#{str} is not a palindrome"
+
+def palin_no_yes str
+    down_str = str.downcase
+    reverse_str = str.downcase.reverse!
+    if down_str == reverse_str
+        "#{str} is a palindrome"
+    else    
+        "#{str} is not a palindrome"
+    end
+end
+
```

Comment:
Good work chaining methods! The `!` at the end of `.reverse` is not necessary! For an additional challenge, how could you refactor this to make it cleaner and more efficient?